lib/plugins/aws/lib/naming.js

```diff
@@ -0,0 +1,174 @@
+'use strict';
+
+const _ = require('lodash');
+
+module.exports = {
+  /*
+   * General Name Normalization
+   */
+  normalizeName(name) {
+    return `${_.upperFirst(name)}`;
```

Comment:
I had problems with `_.upperFirst()` recently in the API Gateway refactoring by @dougmoscrop 

For me `_.capitalize()` was more reliable...